### PR TITLE
TUA-922 Verify that 'Керівник' can add and delete 'Лого', 'Обкладинка' photo

### LINKED ISSUE #185: TUA-922 upload photo in AddClubPopUpStepThree as Manager

### PR DESCRIPTION
**Pull request by Natalia62**

### CODE REVIEW

**Comment by lhalam** [comment]
Conflicting files
src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerTest.java

File: src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerTest.java [hunk 1]
```diff
@@ -32,13 +35,18 @@ public class AddClubPopUpWithManagerTest extends LoginWithManagerTestRunner {
...
+    private WebDriverWait wait;
+
     private AddClubPopUpSider sider;
 
     @BeforeMethod
     public void addClubPopUpTestPrecondition() {
         addClubPopUpComponent = homePage.header.addClubButtonClick();
         stepOne = addClubPopUpComponent.getStepOneContainer();
         softAssert = new SoftAssert();
+        wait = new WebDriverWait(driver, Duration.ofSeconds(30));
```
Comment (s-hrynus) [inline src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerTest.java]: Why did you add here wait?
Oksana wrote wait methods. You can find them here UploadedImgComponent.java

**Review by s-hrynus — CHANGES_REQUESTED** [review]

File: src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerTest.java (hunk 1 above)
Comment (Natalia62) [reply src/test/java/com/academy/ui/addClub/AddClubPopUpWithManagerTest.java]: It's Juliya method. I merged from her branch. I don't use it in my TUA-922
